File: PycharmProjects/introduccion/estructurasdecontrol/numero_menor.py
#vamos a introducir dos numeros por teclado y nos va ha imprimir el menor

num1 = int(input("numero 1:"))
num2 = int(input("numero 2:"))
if num1 < num2:
    print("el numero menor es:", num1)
elif num2 < num1:
    print("el numero menor seria ",num2)
else:
    print("son iguales")

File: eclipse/ejercicio 4/src/estructuracontrol/NumeroMenor.java
package Estructuracontrol;

import java.util.Scanner;

public class NumeroMenor {
	 
	 public static void main(String arg[])
	 {
	 

		 Scanner entrada = new Scanner (System.in);
		 //escrbimos dos numeros y el programa nos mostrara el menor 
		 
		 System.out.println("escribe un numero1:");
		 int num1 = entrada.nextInt();
		 System.out.println("escribe un numero2:");
		 int num2 = entrada.nextInt();
		 
		 if(num1 < num2)
		    System.out.println("el numero menor es: " + num1);
		    
		 else if (num2<num1)
			System.out.println("el numero menor es:  " + num2);
			
			else 
				System.out.println("los numeros son iguales");
	 }	 
		
} 
